ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/FSharpComment.cs

```diff
@@ -92,4 +92,17 @@ public DocComment([NotNull] TokenNodeType nodeType, [NotNull] string text) : bas
     {
     }
   }
+
+  public class XmlDocBlock : FSharpCompositeElement
+  {
+    [NotNull] private readonly CompositeNodeType myNodeType;
+
+    public XmlDocBlock([NotNull] CompositeNodeType nodeType)
+    {
+      myNodeType = nodeType;
```

Comment:
Maybe we should just return `XML_DOC_BLOCK` singleton instead of storing references to it.

Reply:
JFYI I've just checked and found that other `IDocCommentBlock` implementations do exactly that.